BugsnagBundle.php

```diff
@@ -12,4 +14,16 @@ class BugsnagBundle extends Bundle
      * @return string
      */
     const VERSION = '1.0.0';
+
+    /**
+     * {@inheritdoc}
```

Comment:
The description here would literally be "Builds the bundle." - is this really helpful? Anyone who isn't already familiar with Symfony and what this file is for, would not really be looking at it.